docs/can-guides/experiment/setting-up-canjs.md

```diff
@@ -97,7 +97,7 @@ __Note:__ if you see dozens of errors in your console, you may need to set `syst
 StealJS’s [Moving to Production](https://stealjs.com/docs/StealJS.moving-to-prod.html)
 guide has instructions for how to create a production build.
 
-## webpack and npm
+## Webpack and npm
```

Comment:
This change is wrong; [“webpack” is correct](https://webpack.js.org/branding/#the-webpack-name).